tidb-cloud/data-service-postman-integration.md

```diff
@@ -0,0 +1,93 @@
+
+    - The target cluster is selected.
+    - The endpoint path and request method are configured.
+    - The SQL statements are written.
+
+- An [API key](/tidb-cloud/data-service-api-key.md#create-an-api-key) for the Data App.
+
+## Step 1. Import your Data App to Postman
+
+To import your Data App to Postman, take the following steps:
+
+1. In the [TiDB Cloud console](https://tidbcloud.com), navigate to the [**Data Service**](https://tidbcloud.com/console/data-service) page of your project.
+2. In the left pane, click the name of your target Data App to view its details.
+3. In the upper-right corner of the page, click **Run in Postman**. A dialog for import instructions is displayed.
+
+    > **Note:**
+    >
+    > - If a Data App does not have any endpoint with the target cluster, path, request method, and SQL statements configured, **Run in Postman** is unclickable for the Data App.
+    > - For a Chat2Query Data App, **Run in Postman** is unavailable.
+
+4. Follow the steps provided in the dialog for the Data App import:
+
+    1. Depending on the Postman version you want to use, click **Run in Postman for Web** or **Run in Postman Desktop** to open your Postman workspaces, and then select your target workspace in Postman.
+
+        - If you have not logged into Postman, follow the on-screen instructions to log into Postman first.
+        - If you clicked **Run in Postman Desktop**, follow the on-screen instructions to open the Postman desktop app.
+
+    2. On the page of your target workspace in Postman, click **Import** in the left navigation menu.
+    3. Copy the Data App URL in the TiDB Cloud dialog, and then paste the URL to Postman for the import.
+
+5. After you paste the URL, Postman imports the Data App automatically as a new [collection](https://learning.postman.com/docs/collections/collections-overview). The name of the collection is in the `TiDB Data Service - <Your App Name>` format.
+
+    In the collection, the deployed endpoints are grouped in the **Deployed** folder and the undeployed endpoints are grouped in the **Draft** folder.
+
+## Step 2. Configure your Data App API key in Postman
+
+Before running the imported Data App in Postman, you need to configure the API key for the Data App in Postman as follows:
+
+1. In the left navigation menu of Postman, click `TiDB Data Service - <Your App Name>` to open a tab for it on the right side.
+2. Under the `TiDB Data Service - <Your App Name>` tab, click the **Variables** tab.
+3. In the variable table, enter the public key and private key for your Data App in the **Current value** column.
+4. In the upper-right corner of the `TiDB Data Service - <Your App Name>` tab, click **Save**.
+
+## Step 3. Run Data App in Postman
+
+To run your Data App in Postman, take the following steps:
+
+1. In the left navigation pane of Postman, expand the **Deployed** or **Draft** folder, and then click your endpoint name to open a tab for it on the right side.
```

Comment:
In the left navigation pane of Postman, expand the **Deployed** or **Draft** folder, and then select your endpoint by clicking on the name in order to open a working tab for it on the right side.

Reply:
Not sure if this change is necessary as the original one seems more concise?